include/zephyr/bluetooth/iso.h

```diff
@@ -739,6 +751,22 @@ int bt_iso_chan_get_info(const struct bt_iso_chan *chan,
  */
 enum bt_iso_chan_type bt_iso_chan_get_type(const struct bt_iso_chan *chan);
 
+/** @brief Get ISO transmission timing info
+ *
+ *  @details Reads timing information for transmitted ISO packet on CIS or BIS.
+ *           The HCI_LE_Read_ISO_TX_Sync HCI command is used to retrieve this information
+ *           from the controller.
+ *
+ *  @note An SDU must have already been successfully transmitted on the CIS or BIS
+ *        for this function to return successfully.
```

Comment:
Does this mean that the function will return an error? 
If not: should it? 

I guess we can use the existing packet sequence number  in the `bt_iso_chan` (improved upon in https://github.com/zephyrproject-rtos/zephyr/pull/44117) to verify whether something has been sent on an ISO channel.

Reply:
The HCI command will return an error in this case. Adding a separate check in host level is possible, but I don't know if that gives any benefit

Reply:
Generally if the host can know that a HCI command will be rejected, we should perform the check in the host (and of we can avoid having checks in both host and controller with the `CONFIG_BT_CTLR_PARAM_CHECK`).

Reply:
I understand the sentiment, but I'm not convinced the added complexity makes sense in this case.
For example, by adding a check along the lines of `iso_conn->iso.seq_num != 0` I would have to consider if, after #44117 , it would be valid application behavior to consistently send SDUs with sn=0

Reply:
I added a check for the connected state, but not yet using seq_num. I can add a seq_num check if you prefer

Reply:
No I think it is fine for now. In any case, the error message from the controller will be propagated to the caller of this function :)